components/AdditionalDatasheetEditor.tsx

```diff
@@ -282,7 +292,7 @@ export function AdditionalDatasheetEditor() {
             editable: true,
             flex: 1,
             type: 'number',
-            headerAlign: 'left',
+            headerAlign: 'center',
```

Comment:
Hmmm I can see that this would look better because the left aligned header isn't in line with the input field, however I think we should keep this as `left` for consistency.

What we could improve is the spacing around the input field though, this would give more room for the input and make the left alignment look correct. You should be able to simply apply `mx: 0` like here: https://github.com/kausaltech/nzc-data-studio/blob/056506fe22c5e97f4fa9cb20a9ea732d94d711ee/components/DatasheetEditor.tsx#L431

| Baseline data editor | Additional data editor |
|--|--|
| <img width="568" alt="image" src="https://github.com/user-attachments/assets/d6f2bd9d-b474-4a92-bdba-30535bdde2e7"> | <img width="901" alt="image" src="https://github.com/user-attachments/assets/864dd87f-569d-4276-a401-a311f7f6f7b7"> |